### PR TITLE
Update overview.md for void

### PR DESCRIPTION
**Pull request by biopsin**
- add pango-devel to dependencies
- add preferable notice for xbps-src

### CODE REVIEW

File: doc/overview.md [hunk 1]
```diff
@@ -153,9 +153,11 @@ Development tools using the GCC C++ compiler:
 Build dependencies:
 
 ```
-# xbps-install fontconfig-devel libjpeg-turbo-devel libXext-devel libpng-devel libXft-devel libXpm-devel libXrandr-devel libXinerama-devel
+# xbps-install fontconfig-devel libjpeg-turbo-devel libXext-devel libpng-devel libXft-devel libXpm-devel libXrandr-devel libXinerama-devel pango-devel
 ```
 
+However its preferable to use xbps-src to build from template
```
Comment (pekdon) [inline doc/overview.md:159]: Would you mind adding an example on how to do this while at it? (If it isn't too lengthy that is)

Comment (biopsin) [reply doc/overview.md:159]: Guess I could add url for the 
quick-start guide?
https://github.com/void-linux/void-packages#quick-start

Comment (pekdon) [reply doc/overview.md:159]: to build from template, see the [quick-start](https://github.com/void-linux/void-packages#quick-start) for details probably would do the trick yeah.

**Comment by pekdon** [comment]
Thanks for the PR!

**Pull request merged**